camelot/handlers.py

```diff
@@ -83,8 +92,8 @@ def _save_page(self, filename, page, temp):
         """
         with open(filename, 'rb') as fileobj:
             infile = PdfFileReader(fileobj, strict=False)
-            if infile.isEncrypted:
-                infile.decrypt('')
```

Comment:
@rbares Should we change the default value for password to an empty string for this case?

Reply:
And remove the `is not None` check whenever `infile.isEncrypted` is True?

Reply:
Aha! Was initially unsure why `.decrypt('')` was being used. I see from https://github.com/mstamy2/PyPDF2/issues/51 that there are situations where normal PDFs are detected as encrypted, so defaulting to empty string seems sensible. It seems more awkward workarounds are still needed in some cases: https://github.com/mstamy2/PyPDF2/issues/245 , but that feeds into the discussion around which library to use going forward.

Attempting to decrypt with None leads to ``TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'`` rather than the more helpful ``PdfReadError: File has not been decrypted``, so I would be inclined to keep the None checks throughout.

Are you happy to change the defaults or shall I?

Reply:
> Attempting to decrypt with None leads to TypeError: unsupported operand type(s) for +: 'NoneType' and 'str' rather than the more helpful PdfReadError: File has not been decrypted, so I would be inclined to keep the None checks throughout.

But the `TypeError` won't be there if we change the default value of password from `None` to an empty string right? I can change it.

Reply:
Unless someone passes it through explicitly to mean no password is provided, which could be seen as cleaner use of an API. Would almost be nicer to change None to empty string if encrypted rather than making that visible at the interface, but will defer to your judgement.

Reply:
Yes, that is what I was proposing, I think `password=None` at the interface level makes sense, we can set password to an empty string when None is passed. Changing it.